src/common/utils.ts

```diff
@@ -162,3 +162,19 @@ export function truncateString(str: string, maxLength: number) {
   }
   return str.slice(0, maxLength) + '...';
 }
+
+export function getTicker(name: string) {
```

Comment:
Maybe add a comment with some example examples. This is also a great place to add some simple unit tests